src/include/daos/common.h

```diff
@@ -453,6 +453,7 @@ daos_errno2der(int err)
 	case EFAULT:
 	case ENXIO:
 	case ENODEV:
+	case EIO:		return -DER_IO;
```

Comment:
Is it intentional to convert EFAULT, ENXIO and ENODEV to DER_IO? If you just want EIO to be converted to DER_IO, you'd move this line above the "case DEFAULT:"

Reply:
oh yes , I did not looked for the C switch case and think it's like Python.